docs/website/blog/2022-11-29-odo-v3.3.0.md

```diff
@@ -31,7 +31,7 @@ Learn more about the experimental mode in [this doc](../docs/user-guides/advance
 <iframe width="560" height="315" src="https://www.youtube.com/embed/EmN_hGkOHX4?list=PLGMB2PY4SNOr5pjbPTdcWsIrOezCPXWdX" title="odo dev using podman" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
 
 You can now use`odo dev` to run your application on Podman. `odo` leverages the benefit of Podman to work seamlessly with Kubernetes YAML and replicate them to containers locally.
-This is an experimental feature and can be used with `--run-on` flag.
+This is an experimental feature and can be used with `--platform` flag.
```

Comment:
I think we should keep `run-on` in this blog post, as it is specific and relevant to `3.3.0`